Update upgrade-and-coexistence-of-skypeforbusiness-and-teams.md (#3296)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/upgrade-and-coexistence-of-skypeforbusiness-and-teams.md

When some of your users are ready to use only Teams for their day-to-day communications and collaboration needs, you can start upgrading these users to Teams by enabling **Teams Only** mode for them.

> If it’s not feasible for your whole organization to move to Teams, you can start by piloting Teams alongside Skype for Business in **Islands** coexistence mode. With the additional coexistence modes (i.e. **Skype for Business with Teams collaboration** and **Skype for Business with Teams collaboration and meetings**) available, you can start adopting Teams as a group collaboration solution first while keeping Skype for Business as your organization’s unified communications solution. That is Microsoft’s recommended path for customers using Skype for Business Server (on-premises or hybrid) and customers with significant complexity whose trajectory to Teams will include a long coexistence period.

Reviewer: ```suggestion
If it’s not feasible for your whole organization to move to Teams, you can start by piloting Teams alongside Skype for Business in **Islands** coexistence mode. With the additional coexistence modes (i.e. **Skype for Business with Teams collaboration** and **Skype for Business with Teams collaboration and meetings**) available, you can start adopting Teams as a group collaboration solution first while keeping Skype for Business as your organization’s unified communications solution. We recommend this path for customers using Skype for Business Server (on-premises or hybrid) and customers with significant complexity whose trajectory to Teams will include a long coexistence period.
```
